src/main/io/osd.c

```diff
@@ -1600,4 +1608,56 @@ void osdUpdate(timeUs_t currentTimeUs)
 #endif
 }
 
+uint8_t getOSDprofile(void)
+{
```

Comment:
It would be cleaner to keep `OsdProfile` as integer and generate mask when needed

Otherwise you may use `__builtin_ffs`

Reply:
store `osdProfile` as profile index (0,1,2,3) and add `osdProfileMask` (or compute it on the fly when needed)

(or: https://stackoverflow.com/questions/3465098/bit-twiddling-which-bit-is-set)